beacon-chain/rpc/prysm/beacon/BUILD.bazel

```diff
@@ -18,12 +18,15 @@ go_library(
         "//beacon-chain/rpc/eth/helpers:go_default_library",
         "//beacon-chain/rpc/eth/shared:go_default_library",
         "//beacon-chain/rpc/lookup:go_default_library",
```

Comment:
This file is missing somethings. Please run `bazel run //:gazelle`

Reply:
done